instance/models/instance.py

```diff
@@ -75,6 +75,18 @@ class InstanceState(ResourceState):
     """
     A [finite state machine] state describing an instance.
     """
+    # A state is "steady" if we don't expect it to change.
+    # This information can be used to:
+    # - delay execution of an operation until the target server reaches a steady state
+    # - raise an exception when trying to schedule an operation that depends on a state change
+    #   while the target server is in a steady state
+    # Steady states include: Status.Running, Status.ConfigurationFailed, Status.Error, Status.Terminated
+    is_steady_state = False
+
+    # An instance is healthy if it has Status.Running.
+    # This information can be used to delay execution of an operation
+    # until the target instance has reached this status.
+    is_healthy_state = False
```

Comment:
I think that New, WaitingForServer, ConfiguringServer, and Running should all be considered healthy states, since they're all part of a normal/expected workflow. "Unhealthy" should mean a state that is not expected and indicates that there is a problem.

In other words, we should be able to use this in the UI to highlight error states when is_healthy_state is false, and we don't want to highlight a "ConfiguringServer" instance in red as though there's an error.